Another round of minor specification review fixes (#1126)
riscv/riscv-cheri: src/cheri/riscv-hybrid-integration.adoc

==== Observing the {cheri_mode_name}

The effective {cheri_mode_name} cannot be determined just by reading the <<p_bit>> from the <<pcc>> since it also depends on the execution environment.
The following code sequence demonstrates how a program can observe the current, effective {cheri_mode_name}.

> It will write, to `x1`, the value `{CAP_MODE_VALUE}` for {cheri_cap_mode_name} (where <<AUIPC_CHERI>> derives a valid capability from the <<pcc>>)

Reviewer: ```suggestion
It will write, to `x1`, the value `1` for {cheri_cap_mode_name} (where <<AUIPC_CHERI>> derives a valid capability from the <<pcc>>)
```

> It will write, to `x1`, the value `{CAP_MODE_VALUE}` for {cheri_cap_mode_name} (where <<AUIPC_CHERI>> derives a valid capability from the <<pcc>>)

Author: This is wrong, the fixed 1/0 was correct since we are talking about the return value of ytagr

> or `{INT_MODE_VALUE}` for {cheri_int_mode_name} (where `auipc` reverts to its standard behavior and writes an integer result with a clear {ctag}):

Reviewer: ```suggestion
or `0` for {cheri_int_mode_name} (where `auipc` reverts to its standard behavior and writes an integer result with a clear {ctag}):
```

The effective {cheri_mode_name} cannot be determined just by reading the <<p_bit>> from <<pcc>> since it also depends on the execution environment.
The following code sequence demonstrates how a program can observe the current, effective {cheri_mode_name}.

> It will write, to `x1`, the value `1` for {cheri_cap_mode_name} (where <<AUIPC_CHERI>> derives a capability with a set {ctag} from <<pcc>>)

Reviewer: ```suggestion
It will write, to `x1`, the value `1` for {cheri_cap_mode_name} (where <<AUIPC_CHERI>> derives a valid capability from <<pcc>>)
```
